workshops/flappy_learning/README.md

```diff
@@ -27,8 +27,8 @@ Here is the [starter project](https://repl.it/@iMega/flappy-learning-starter#cod
 
 _^ You will see something like this._
 
-## NeuroEvolution Initialization
-Let's begin our coding! We start from our first variable declaration for the Neuroevolution, let's give it a name, like Neuvol and game variable for the game function. Then we initialize the start function.
+## First Line of Code
```

Comment:
This title doesn't make sense to me.

Reply:
Any suggestions for me? What could be a cooler and sensible title?

Reply:
please check it here #1599